saleor/order/events.py

```diff
@@ -95,6 +95,53 @@ def draft_order_removed_products_event(
     )
 
 
+def invoice_requested_event(
```

Comment:
This will fail if mutation will be triggered by App Token.

Reply:
All your events methods will fail

Reply:
We had the same issue with CSV import/export. What approach did we agree on there? I think we decided to add it later and move the app support to another epic?

Reply:
Just added a simple fail-safe for that which we use in other functions in that file

Reply:
Yes, I created an epic for that.